x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/curation/curation.tsx

```diff
@@ -38,20 +40,27 @@ export const Curation: React.FC<Props> = ({ curationsBreadcrumb }) => {
 
   return (
     <>
-      <SetPageChrome trail={[...curationsBreadcrumb, curation.queries.join(', ')]} />
+      <SetPageChrome trail={[...curationsBreadcrumb, queries.join(', ')]} />
       <EuiPageHeader
         pageTitle={MANAGE_CURATION_TITLE}
         /* TODO: Restore defaults button */
         responsive={false}
       />
 
-      {/* TODO: Active query switcher / Manage queries modal */}
+      <EuiFlexGroup alignItems="flexEnd" gutterSize="xl" responsive={false}>
+        <EuiFlexItem>
+          <ActiveQuerySelect />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <ManageQueriesModal />
```

Comment:
The pop-out modal design of this combined with the ActiveQuerySelect above seems strange to me tbh. Messing around with this in the UI (both here and in feels hard to grok how curations -> queries -> curated documents related to each other.  In my head I'd expect the list of queries to be the main content of this page, and then each query linking to a subview to manage it.  I'm not sure I agree with the organization for this view to have the selector/detail view as the main content, and the list be in a modal. I recognize this is similar to how our current UX in `ent-search` works, and I'm happy the selector got moved out of the modal, but maybe its worth removing the modal entirely. 

Rather than re-organize this whole view though,  I'd consider wrapping this custom component out for an `EuiComboBox`, inline on the page, above the query editor.  I think that would better re-enforce "here are the queries you've selected, now you can select one below to edit it's details on this page.  If we're worried about friction of deleting a query, we could add a confirm dialogue as we've done elsewhere for deletions.  As a bonus, this has a smaller surface area for coverage!

Reply:
Hmm, I think I'm struggling to picture this UI/UX with the [ComboBox](https://elastic.github.io/eui/#/forms/combo-box) - I can't quite visualize how you select the active query in combination with the combobox. So it's a dropdown and then there's a 2nd dropdown for the active query? I guess it's hard for me to picture how that might look next to each other.

I do agree though it does look like the combobox could replace the rows UI/UX + getting rid of an extra modal is always nice! I think @daveyholler might be making another pass/polish round on the Curation page at some point, definitely worth pitching this to him.

> In my head I'd expect the list of queries to be the main content of this page, and then each query linking to a subview to manage it

I just want to be clear on this though, the main action/purpose of this page is promoting & hiding documents for the curation, not managing the queries. The list of promoted / hidden documents applies to **all** queries in the array, those do not change if you change the active query. Changing the active query just lets you see different organic results for that query (although you can manually search for those anyway).

Reply:
From Davey re: Combobox usage

> Sweet. I like the idea on the surface. I’ll play around with it and see how it feels. Definitely a solid idea for the polish pass

🎉